Binary files angband-3.0.6/angband-3.0.6-src.tar.gz and angband-3.0.6-borg/angband-3.0.6-src.tar.gz differ
diff -Nru --strip-trailing-cr angband-3.0.6/src/config.h angband-3.0.6-borg/src/config.h
--- angband-3.0.6/src/config.h	2005-02-08 05:16:36.000000000 -0800
+++ angband-3.0.6-borg/src/config.h	2010-10-19 07:02:14.000000000 -0700
@@ -141,7 +141,7 @@
 /*
  * OPTION: Hack -- Compile in support for "Borg mode"
  */
-/* #define ALLOW_BORG */
+#define ALLOW_BORG
 
 
 /*
@@ -220,7 +220,7 @@
 /*
  * OPTION: Allow "Borgs" to yield "high scores"
  */
-/* #define SCORE_BORGS */
+#define SCORE_BORGS
 
 /*
  * OPTION: Allow "Cheaters" to yield "high scores"
